playbooks/bootstrap.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Check remote connections
+  hosts: "{{ edpm_override_hosts | default('all', true) }}"
+  gather_facts: false
+  tasks:
+    - name: Wait 300 seconds, but only start checking after 10 seconds
```

Comment:
I believe this should be parametrized, ideally in a way that allow you to skip this check if you want, or maybe set the time to be slightly long or shorter.

Reply:
Why would you skip it? 10 seconds initial wait is not too much.  Also, we  don't want to wait longer than 5 mins for sure. Everything does not need to be parameterized.

Reply:
Maybe because I don't want to wait at all. Or maybe I'm on network with bad latency. Adding parameter with default is easy enough. There is no reason to hard code this.

Reply:
I don't think the reason for requesting change is justified here. I've never seen customers request that to be changed in TripleO. Anyway, I've updated it now.